File: 백준/Silver/26069. 붙임성 좋은 총총이/붙임성 좋은 총총이.py
import sys

input = sys.stdin.readline
N = int(input())
meet_chongchong = False
dance = set()
count = 0
for _ in range(N) :
    M,K = input().split()

    if K == "ChongChong" or M == "ChongChong" :
        meet_chongchong = True
        dance.add(K)
        dance.add(M)
        continue

    if meet_chongchong and (M in dance or K in dance) :
        dance.add(K)
        dance.add(M)

print(len(dance))

File: 백준/Silver/26069. 붙임성 좋은 총총이/붙임성 좋은 총총이.js
const fs = require("fs");
const filePath = process.platform === "linux" ? "/dev/stdin" : "./input.txt";
const [n, ...input] = fs.readFileSync(filePath).toString().split("\n");

let people_set = new Set();
people_set.add('ChongChong');

for(let i = 0;i < n;i++){
    const [n,m] = input[i].split(" ");
    if(people_set.has(n) || people_set.has(m)) {
        people_set.add(n)
        people_set.add(m)
    }
}
console.log(people_set.size)